api/spec/requests/spree/api/v2/storefront/products_spec.rb

```diff
@@ -451,6 +451,35 @@
           end
         end
       end
+
+      context 'sorting by sku' do
+        let(:skus_array) { json_response['data'].map { |p| p['attributes']['sku']} }
+        let(:available_products_with_master) { products + [product_with_option, in_stock_product, not_backorderable_product, product_with_property] }
+
+        context 'ascending order' do
+          before { get '/api/v2/storefront/products?sort=sku' }
```

Comment:
@szymoniwacz let's also add stest 

1. where we include variants
2. where we sort by SKU and another param like updated at

Reply:
@damianlegawiec Spree::BaseSorter is designed to sort only by one param.